cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/RatioTapChangerConversion.java

```diff
@@ -34,7 +34,8 @@ public RatioTapChangerConversion(PropertyBag rtc, Context context) {
         lowStep = rtc.asInt("lowStep");
         highStep = rtc.asInt("highStep");
         neutralStep = rtc.asInt("neutralStep");
-        position = fromContinuous(p.asDouble("SVtapStep", neutralStep));
+        int defaultStep = rtc.asInt("normalStep", neutralStep);
+        position = fromContinuous(rtc.asDouble("step", rtc.asDouble("SVtapStep", defaultStep)));
```

Comment:
If we use the step from the SSH for the tap position as first option, we will not build an IIDM state that is consistent with the observed voltages copied from the SV data. If the given data is a power flow solution the voltages have been obtained for the tap position that is given in the SV data. If we use the tap position from SSH and recalculate flows, they will not match. Needs rethinking.